docs/deployment/nats/leaf-nodes.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Leaf Nodes & Sidecars'
+description: 'A common way to for hosts to connect securely to a lattice'
+sidebar_position: 2
+---
+
+[NATS leaf nodes](https://docs.nats.io/nats-server/configuration/leafnodes) are a flexible way to extend NATS infrastructure. Leaf nodes enable segmentation or isolation of traffic and/or security boundaries.
+
+wasmCloud can use leaf nodes to simplify lattice configuration. A common pattern involves starting a NATS leaf node alongside a wasmCloud host on a system (virtual or otherwise). The leaf node is configured with credentials for authenticating with an external NATS cluster. By delegating the NATS authentication to the leaf node, the wasmCloud host can connect to the leaf node anonymously.
+
+This type of "anonymous localhost" connection is commonly used in the [sidecar](https://docs.microsoft.com/en-us/azure/architecture/patterns/sidecar) pattern, which is why we've labelled this this "sidecar leaf node" pattern.
+
+This segmentation enables bridging disparate infrastructure. Using leaf nodes at the "edges" or boundaries of your infrastructure allows the lattice to expose a single, flat topology, while NATS and leaf nodes take care of all the hard work of optimizing traffic patterns for the interest graph across an arbitrarily complex network. Leaf nodes can be used to string together disparate infrastructures, multiple private cloud environments, etc. Their simplicity is key; check out the NATS documentation to learn more.
```

Comment:
include NATS doc link? not sure what page

Reply:
Intent was to refer to the link at the start of this page, will make clear